src/idom/utils.py

```diff
@@ -56,6 +59,24 @@ def __repr__(self) -> str:
         return f"{type(self).__name__}({current})"
 
 
+def vdom_to_html(value: VdomDict) -> str:
+    """Convert a VDOM dictionary into an HTML string
+
+    Only the following keys are translated to HTML:
+
+    - ``tagName``
+    - ``attributes``
+    - ``children`` (must be strings or more VDOM dicts)
+    """
+    temp_root = etree.Element("__temp__")
+    _add_vdom_to_etree(temp_root, value)
+    return (
+        cast(bytes, tostring(temp_root)).decode()
```

Comment:
The way this return is written makes it harder to read. We should use some variables for improved clarity.

```python
html = cast(bytes, tostring(temp_root)).decode()

# strip out temp root <__temp__> element
return html[10:-11]
```